src/e2e/puppeteer/helpers/simulateDragAndDrop.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'puppeteer'
+
+/** Simulates Drag and Drop functionality in Puppeteer. */
+const dragAndDrop = async (page: Page, selectorDrag: string) => {
```

Comment:
This function can be split into a function for general drag-and-drop (of any element) and a function for dragging and dropping a thought.

```ts
dragAndDrop(page: Page, selectorDrag: string, selectorDrop: string)

dragAndDropThought(
  page: Page, 
  sourceValue: string, 
  destValue: string, 
  { position: 'after' | 'before' | 'child' }
)
```